src/main/java/au/gov/ga/geodesy/support/marshalling/moxy/GeodesyMLMoxy.java

```diff
@@ -118,6 +119,7 @@ public void marshal(Object x, Writer writer) throws MarshallingException {
     public <T> JAXBElement<T> unmarshal(Reader reader, Class<T> type) throws MarshallingException {
         try {
             JAXBElement<?> element = (JAXBElement<?>) createUnmarshaller().unmarshal(reader);
+            new GMLPropertyTypeResolver(element.getValue());
```

Comment:
Why do you create this but don't use it?  It isn't a Singleton or seem to in any way exist after this method finishes.